packages/styles/scss/patterns/blocks/callout-data/_callout-data.scss

```diff
@@ -28,6 +28,8 @@
         &__source {
           @include carbon--type-style('body-short-01', true);
 
+          width: 90%;
+          max-width: 640px;
```

Comment:
should `carbon--mini-units` be used here?

Reply:
@ariellalgilmore change applied: https://github.com/carbon-design-system/ibm-dotcom-library/pull/3025/commits/b6596afa9f813c3dc944e79e82a261afa119fcf8